chain/events/filter/index.go

```diff
@@ -404,3 +412,118 @@ func (ei *EventIndex) PrefillFilter(ctx context.Context, f *EventFilter) error {
 
 	return nil
 }
+
+func (ei *EventIndex) RunMigration(from, to int) error {
```

Comment:
I tested the migration on my local database successfully. Was able to verify that we kept returning data correctly through the Eth JSON-RPC. Considered adding a test, but because this is very temporary code I felt it wasn't worth it. Can reconsider if folks feel strongly about this.